config.go

```diff
@@ -49,6 +48,7 @@ var (
 	btcwalletHomeDir   = btcutil.AppDataDir("btcwallet", false)
 	btcdHomedirCAFile  = filepath.Join(btcdHomeDir, "rpc.cert")
 	defaultConfigFile  = filepath.Join(btcwalletHomeDir, defaultConfigFilename)
+	defaultPubPassphrase = "public"
```

Comment:
It doesn't belong in the main package.  I would add it to waddrmgr and export it so the wallet package can use it.